src/components/DigitalCredentials/dialogs/CredentialRevokedDialog.vue

```diff
@@ -63,26 +64,26 @@ export default class CredentialRevokedDialog extends Vue {
 
 .close-btn {
   position: absolute;
-  right: 2rem;
+  right: $px-32;
 }
 
 .action-btn {
   min-height: 44px;
 }
 
 .warning-title {
-  font-size: 1.5rem;
+  font-size: $px-24;
   font-weight: bold;
   color: $gray9;
   text-align: center;
-  line-height: 2.125rem;
+  line-height: $px-34;
```

Comment:
Since line height is tied to font size, it makes sense to use these CSS variables. 👍